shell/platform/android/embedding_bundle/build.gradle

```diff
@@ -48,6 +48,8 @@ android {
     embedding "androidx.lifecycle:lifecycle-common:$lifecycle_version"
     embedding "androidx.lifecycle:lifecycle-common-java8:$lifecycle_version"
 
+    embedding "com.google.android.play:core:1.8.0"
```

Comment:
ah of course. This makes sense. Though we should try to clarify this high level question early on. Could you take a hello_world app and build it with and without this in release mode (thus R8'ed)? We should compare the APK sizes of the engine with and without this dependency to see if this is an acceptable increase. 

We can also drag and drop the old and new APK into Android Studio, introspect the .dex and look at all the new Java classes that's brought in to see if it makes sense or we need to tweak the default R8 rules more.

Reply:
From what I can tell form the docs, it looks like this package is pretty lightweight. I don't expect it to add much, though I'll also run the experiment to put a number on it.

Reply:
We should test. One possible case is that that the package itself has very little code but declares other transitive Gradle dependencies.